tests/test-common.js

```diff
@@ -3,9 +3,70 @@
  */
 'use strict';
 
+const chai = require('chai');
+const {expect} = chai;
+
 module.exports = async function({Secp256k1KeyPair}) {
 
   describe('secp256k1-key-pair', () => {
+
+    describe('Secp256k1KeyPair', ()=>{
+      it('has working constructor', async () => {
+        const x = new Secp256k1KeyPair({
+          id: undefined,
+          controller: undefined,
+          type: 'EcdsaSecp256k1VerificationKey2019',
+          privateKeyBase58: '4HvXrvNBrmN5tUCwcjVWRpQG32CtuLvZ12xVf5rv8r1F',
+          publicKeyBase58: '231cRx1fhyNzrdj9i3UseKm1ApgMwyDLbKtJJH5AacEwL'
+        });
+        x.should.be.a('object');
+        expect(x.privateKeyBase58).to.equal('4HvXrvNBrmN5tUCwcjVWRpQG32CtuLvZ12xVf5rv8r1F');
+        expect(x.publicKeyBase58).to.equal('231cRx1fhyNzrdj9i3UseKm1ApgMwyDLbKtJJH5AacEwL');
+      });
+
+      it('has static factory method from', async () => {
+        const x = await Secp256k1KeyPair.from({
+          id: undefined,
+          controller: undefined,
+          type: 'EcdsaSecp256k1VerificationKey2019',
+          privateKeyBase58: '4HvXrvNBrmN5tUCwcjVWRpQG32CtuLvZ12xVf5rv8r1F',
+          publicKeyBase58: '231cRx1fhyNzrdj9i3UseKm1ApgMwyDLbKtJJH5AacEwL'
+        });
+        x.should.be.a('object');
+        expect(x.privateKeyBase58).to.equal('4HvXrvNBrmN5tUCwcjVWRpQG32CtuLvZ12xVf5rv8r1F');
+        expect(x.publicKeyBase58).to.equal('231cRx1fhyNzrdj9i3UseKm1ApgMwyDLbKtJJH5AacEwL');
+      });
+
+      it('has static method fingerprintFromPublicKey', async () => {
+        const f0 = Secp256k1KeyPair.fingerprintFromPublicKey({publicKeyBase58: '231cRx1fhyNzrdj9i3UseKm1ApgMwyDLbKtJJH5AacEwL'});
```

Comment:
I believe this is a linting error (80 char max).

Reply:
There is no lint configuration for this repo currently. I adjusted the line length for when one is added.